mobile_app/packages/teigiii_api/doc/FrameStatsDevice.md

```
@@ -0,0 +1,21 @@
+
+## Load the model package
+```dart
+import 'package:teigiii_api/api.dart';
+```
+
+## Properties
+Name | Type | Description | Notes
+------------ | ------------- | ------------- | -------------
+**appVersion** | **String** |  | 
+**buildNumber** | **int** |  | 
+**flavor** | **String** |  | 
```

markdownlint-cli2 (0.23.0)

[warning] Table pipe style
Expected: no_leading_or_trailing; Actual: trailing_only; Unexpected trailing pipe

(MD055, table-pipe-style)